linux/open_vm_tools/pre_ovt_install.yml

```diff
@@ -38,6 +38,50 @@
     - vmtools_is_installed
     - vmtools_install_type != 'package'
 
+- name: "Set fact of open-vm-tools install type"
+  ansible.builtin.set_fact:
+    ovt_install_type: "{{ linux_ovt_install_type | default('package') }}"
+
+- name: "Display open-vm-tools install type"
+  ansible.builtin.debug:
+    msg: "The open-vm-tools install type is {{ ovt_install_type }}"
+
+- name: "Block test case because of missing linux_ovt_tarball_url for open-vm-tools source install"
+  include_tasks: ../../common/skip_test_case.yml
+  vars:
+    skip_msg: >-
+      Skip test case {{ ansible_play_name }} because 'linux_ovt_tarball_url' is not set, which is
+      required when open-vm-tools install type is {{ ovt_install_type }}.
+    skip_reason: "Blocked"
+  when:
+    - ovt_install_type == 'source'
+    - linux_ovt_tarball_url is undefined or not linux_ovt_tarball_url
+
+- name: "Skip test case when open-vm-tools install type is not applicable"
+  include_tasks: ../../common/skip_test_case.yml
+  vars:
+    skip_msg: >-
+      Skip test case {{ ansible_play_name }} because open-vm-tools
```

Comment:
I think "Skip test case {{ ansible_play_name }} when open-vm-tools install type is set to {{ ovt_install_type }}" is more easier for understanding.

Reply:
Updated. Thanks.